scripts/mkstatic.py

```diff
@@ -216,6 +216,13 @@ class API(object):
 !
 module {module}
 
+#if defined MPI_F08
```

Comment:
I guess if the same lines are repeated at all levels then it makes sense to stick these 6 lines in an include file, plus they will need to be repeated in MPI enabled schemes?
Are we down with include files in the CCPP?

Reply:
I would prefer an include file as well, yes